packages/create-next-app/index.ts

```diff
@@ -396,7 +402,8 @@ async function run(): Promise<void> {
         program.app = Boolean(appRouter)
       }
     }
-    if (!process.argv.includes('--turbo') && !process.argv.includes('--no-turbo')) {
+
+    if (!program.turbo) {
```

Comment:
```suggestion
    if (!process.argv.includes('--turbo')) {
```

Reply:
It's a program.turbo since it uses `commandjs`, where `.option` passes value as turbo there

Reply:
e.g. `--empty` to `.option` passes `program.empty` value in [Commander](https://www.npmjs.com/package/commander#common-option-types-boolean-and-value)

Reply:
I got it, thank you, this was the general use, so I suggested this

Reply:
I understand, thank you very much. I suggested it because this was the general usage :+1:

Reply:
Awesome, thank you for reviewing as well 😌